api/client/batch_client.py

```diff
@@ -79,22 +79,33 @@ def get_data(self, url, headers, params=None):
                         url, retry_count, e.response.error))
 
     @gen.coroutine
-    def get_data_points(self, **selection):
-        """Get all the data points for a given selection, which is some or all
-        of: item_id, metric_id, region_id, frequency_id, source_id,
-        partner_region_id. Additional arguments are allowed and ignored.
-        """
+    def get_data_points_generator(self, **selection):
         headers = {'authorization': 'Bearer ' + self.access_token}
         url = '/'.join(['https:', '', self.api_host, 'v2/data'])
         params = lib.get_data_call_params(**selection)
         resp = yield self.get_data(url, headers, params)
         raise gen.Return(json_decode(resp))
 
+    @gen.coroutine
+    def get_data_points(self, **selection):
+        """Get all the data points for a given selection, which is some or all
+        of: item_id, metric_id, region_id, frequency_id, source_id,
+        partner_region_id. Additional arguments are allowed and ignored.
+        """
+        data_points = super(BatchClient, self).get_data_points(**selection)
+        raise gen.Return(data_points)
+
+    def get_df(self):
```

Comment:
here we need add option show_revision=True

Reply:
done